### PR TITLE
DROOLS-3550: [Stunner] Allow the use of different text wrap strategy

### PR DESCRIPTION
**Pull request by danielzhe**
Part of an ensemble:
https://github.com/kiegroup/kie-wb-common/pull/2429
https://github.com/kiegroup/lienzo-core/pull/87

### CODE REVIEW

**Comment by manstis** [comment]
@romartin what exactly is there to test?

File: src/main/java/com/ait/lienzo/client/core/shape/TextBoundsWrap.java [hunk 1]
```diff
@@ -64,9 +64,8 @@ public BoundingBox getWrapBoundaries() {
         return wrapBoundaries;
     }
 
-    public TextBoundsWrap setWrapBoundaries(final BoundingBox boundaries) {
+    public void setWrapBoundaries(final BoundingBox boundaries) {
```
Comment (jomarko) [inline src/main/java/com/ait/lienzo/client/core/shape/TextBoundsWrap.java]: Please add `@Override` annotation

File: src/main/java/com/ait/lienzo/client/core/shape/TextTruncateWrapper.java [hunk 2]
```diff
@@ -45,11 +45,9 @@ public BoundingBox getWrapBoundaries()
         return m_wrapBoundaries;
     }
 
-    public TextTruncateWrapper setWrapBoundaries(final BoundingBox boundaries)
+    public void setWrapBoundaries(final BoundingBox boundaries)
```
Comment (jomarko) [inline src/main/java/com/ait/lienzo/client/core/shape/TextTruncateWrapper.java]: please add `@override` annotation

**Comment by danielzhe** [comment]
@jomarko I just pushed the required changes.

**Comment by danielzhe** [comment]
Jenkins execute full downstream build

**Pull request merged**